src/goto-programs/goto_contractor.cpp

```diff
@@ -339,21 +772,26 @@ ibex::Function *goto_contractort::create_function_from_expr2t(expr2tc expr)
   case expr2t::expr_ids::neg_id:
   {
     h = create_function_from_expr2t(to_neg2t(expr).value);
+    if(h == nullptr)
+      return nullptr;
     f = new ibex::Function(*vars, -(*h)(*vars));
     break;
   }
   case expr2t::expr_ids::symbol_id:
   {
     int index = create_variable_from_expr2t(expr);
-    if(index != CspMap::NOT_FOUND)
+    if(index == CspMap::NOT_FOUND)
     {
-      f = new ibex::Function(*vars, (*vars)[index]);
+      log_error("ERROR: MAX VAR SIZE REACHED");
```

Comment:
log_error() already prints "ERROR", no need to prefix.

Reply:
fixed.